### DISCUSSION TITLE
Code review IM-11

### DISCUSSION THREAD

**Discussion by tomatorepublic** [HowProgrammingWorks/LiveQA#108] [Code review]
Кізь Данило, ІМ-11.
Програма, яка дозволяє вивчати англійські слова прямо в терміналі,
коли у розробника кипить мозок і він хоче перевести свою увагу.
Буду вдячний за рев'ю: https://github.com/tomatorepublic/companion

**Comment by tshemsedinov** [comment]
See: https://github.com/tomatorepublic/companion/issues/1

**Reply by tomatorepublic** [reply]
Все пофіксив та додав нового функціоналу
